docs/source/contents/4_draft_mapping.rst

```diff
@@ -67,6 +79,20 @@ Mapping
 identifier
 ==========
 
++------------------------+------------+-------------------+---------------------------------------------------------------------------------------+
+| Predicate              | Value Type | Range (if needed) | Usage Notes                                                                           |
++------------------------+------------+-------------------+---------------------------------------------------------------------------------------+
+| dbo:isbn               | Literal    |                   | Use for identifiers with type=""isbn                                                  |
++------------------------+------------+-------------------+---------------------------------------------------------------------------------------+
+| dbo:issn               | Literal    |                   | Use for identifiers with type=""issn"                                                 |
```

Comment:
same; small typo `type=""issn"`

Reply:
Thanks for catching this. I've just committed changes for both typos and hopefully not introduced any table spacing issues in the process.